sendgrid-php.php

```diff
@@ -1,3 +1,28 @@
 <?php
-require __DIR__ . '/vendor/autoload.php';
-?>
\ No newline at end of file
+/**
+ * This file is used to load the Composer autoloader if required.
+ */
+
+use SendGrid\Mail\Mail;
+
+// Define path/existence of Composer autoloader
+$composerAutoloadFile = __DIR__ . '/vendor/autoload.php';
+$composerAutoloadFileExists = (is_file($composerAutoloadFile));
+
+// Can't locate SendGrid\Mail\Mail class?
+if (!class_exists(Mail::class)) {
+    // Suggest to load Composer autoloader of project
+    if (!$composerAutoloadFileExists) {
+        //  Can't load the Composer autoloader in this project folder
+        error_log("Composer autoloader not found. Execute 'composer install' in the project folder first.");
+    } else {
+        // Load Composer autoloader
+        require_once $composerAutoloadFile;
```

Comment:
Why the change from `require` to `require_once`? Would this break anything?

Reply:
It's personal preference, but mostly used when loading files which contains functions/classes.
Not breaking because `$composerAutoloadFile` will be loaded anyway, but just once.
No big thing to change back to `require`.